src/main/pom.xml

```diff
@@ -19,6 +19,17 @@
   <name>Main Module</name>
 
   <dependencies>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bc-fips</artifactId>
+      <version>1.0.2.5</version>
```

Comment:
this is a version from Apr 29, 2021 with 2 known vulnerabilities; please use the current 2.1.0 version